app/controllers/widgets_controller.rb

```diff
@@ -1,4 +1,4 @@
-class WidgetsController < ActionController::Base
+class WidgetsController < ApplicationController
```

Comment:
Widget do not need to inherit ApplicationController for the reasons mentioned above